wordpress-mobile/gutenberg-mobile
# [Android Aztec] - Fixes refresh issues on merging blocks

### Description:

Fixes refresh issues in Android when merging blocks.

Moves #167 forward.

### Related PRs:

Aztec RN Wrapper PR: https://github.com/wordpress-mobile/react-native-aztec/pull/67 [Merged]

### Testing:

1. Make sure you have 2 blocks of the same type (paragraph x 2 or heading x2).
2. Place the caret at the beginning of the bottom block.
3. Press backspace.
4. Make sure the resulting block is as expected and refreshed properly.
